tests/unit/meta/meta.ts

```diff
@@ -0,0 +1,189 @@
+import global from '@dojo/core/global';
+import * as registerSuite from 'intern!object';
+import * as assert from 'intern/chai!assert';
+import { v } from '../../../src/d';
+import { ProjectorMixin } from '../../../src/main';
+import { Base as MetaBase } from '../../../src/meta/Base';
+import { WidgetBase } from '../../../src/WidgetBase';
+import { stub } from 'sinon';
+
+let rAF: any;
+
+function resolveRAF() {
+	for (let i = 0; i < rAF.callCount; i++) {
+		rAF.getCall(i).args[0]();
+	}
+	rAF.reset();
+}
+
+registerSuite({
+	name: 'meta base',
+
```

Comment:
No, it's just the way I was asked to do it in another PR I had worked on. I'll remove.